control/installation.xml

```diff
@@ -17,19 +17,17 @@ textdomain="cc-control"
                 <system_role>
                   <id>cc_role</id>
 		  <order config:type="integer">1100</order>
-		  <!--
-		  <additional_dialogs>inst_cc_foo</additional_dialogs>
-		  -->
+                  <additional_dialogs>inst_cc_mode</additional_dialogs>
```

Comment:
wow, I never see it used for ages. So lets see how it will work

Reply:
I tried it, it works :)